### ISSUE TITLE
Warning logs unrelated to my code

### ISSUE THREAD

**Issue by osanseviero** [gradio-app/gradio#1777]
### Describe the bug

I run a gradio app and get some warnings that seem unrelated

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

https://huggingface.co/spaces/osanseviero/test-eur/blob/main/app.py

### Logs

```shell
/usr/local/lib/python3.7/dist-packages/gradio/deprecation.py:40: UserWarning: `optional` parameter is deprecated, and it has no effect
  warnings.warn(value)
/usr/local/lib/python3.7/dist-packages/gradio/deprecation.py:40: UserWarning: `numeric` parameter is deprecated, and it has no effect
  warnings.warn(value)
/usr/local/lib/python3.7/dist-packages/gradio/deprecation.py:40: UserWarning: The 'type' parameter has been deprecated. Use the Number component instead.
  warnings.warn(value)

```
```


### System Info

```shell
Spaces or Colab, 3.0.24
```

### Severity

annoying

**Comment by osanseviero** [comment]
@freddyaboulton Sorry, I just made it public. And no, this is the demo

```python
print(gr.__version__)
iface = gr.Interface(
  fn=predict, 
  inputs=[gr.inputs.Textbox(label="text", lines=3)],
  outputs='text',
  title=title,
  examples=[["Hello! My name is Omar"], ["I like this workshop"]]
)
```

It might be due to using `gr.inputs.Textbox` rather than `gr.Textbox` though. Will change in a sec to try it out.

**Comment by osanseviero** [comment]
Note there is no warning about using `gr.inputs.Textbox` which was something expected as shared in another internal discussion today https://huggingface.slack.com/archives/C02QZLG8GMN/p1657198107532359

**Comment by osanseviero** [comment]
Cool! So then it's related to the slack convo/https://github.com/gradio-app/gradio/issues/1778 . I'll close this issue then.

**Comment by amiruak05** [comment]
Hello, My image recognition gui is returning
 UserWarning: The 'type' parameter has been deprecated. Use the Number component instead.
  warnings.warn(value).

please see code;
inception_net = tf.keras.applications.InceptionV3()

# Set our output results
response = requests.get("https://drive.google.com/file/d/1cBAMe0RtQEPXQOMZewhZ9jDNkmN1h1_K/view?usp=share_link")
labels = response.text.split("\n")

def classify_image(img):
  img_new = img.reshape((-1, 299, 299, 3))
  img_resize = img_new
  img = tf.keras.applications.inception_v3.preprocess_input(img)
  prediction = inception_net.predict(img).flatten()
  return {label[i]: float(prediction[i]) for i in range(2)}

image = gr.inputs.Image()
label = gr.outputs.Label(num_top_classes= 2)

my_project = gr.Interface(fn=classify_image, inputs=image, outputs=label, capture_session = True)

my_project.launch(share = True)
